hybrid/hybrid_simulation.py

```diff
@@ -179,7 +179,10 @@ def __init__(self,
         if 'geothermal' in power_sources.keys():
             raise NotImplementedError("Geothermal plant not yet implemented")
         if 'grid' in power_sources.keys():
-            self.grid = Grid(self.site, power_sources['grid'])
+            if 'grid_source' in power_sources['grid']:
+                self.grid = power_sources['grid']['grid_source']                # User instantiated grid source
```

Comment:
What is this "grid_source" option? Can you please add some documentation?

Reply:
It's the instantiated grid object that can be passed to HybridSimulation, implemented the same way as the instantiated 'pv_plant' object a few lines above that. I'll add documentation.